public/static/docs/glossary.js

```diff
@@ -9,12 +9,9 @@ export default {
       match: ['workspace'],
       desc: `
 Directory containing all your project files. For example raw datasets, source
-code, ML models, etc. A workspace becomes a **DVC project** when
-[\`dvc init\`](/doc/command-reference/init) is run, and
-[DVC-files](/doc/user-guide/dvc-file-format) or stage files are created in it.
-
-Note that [external outputs](/doc/user-guide/managing-external-data) also
-form part of your expanded workspace, technically.
+code, ML models, etc. Note that
+[external outputs](/doc/user-guide/managing-external-data) also form part of
+your expanded workspace.
```

Comment:
@shcheklein please also notice this change over in this PR as we recently discussed. Thanks

Reply:
I really don't like this note about external outputs (even if I suggested i, hah - I'm not sure where does it come from:) ). It would not make any sense to 100% readers. We can even test it with our team. Could recall what was the reason?

Reply:
Sure, I can remove it. That note was already there though, this change removes the 2nd sentence (about becoming a DVC project). I'll test the external outputs sentence with the team meanwhile...

Reply:
test and ask if they are able to understand this :) Do you remember why was it added?

Reply:
Just Ruslan answered, he said it makes sense to him. I think they will understand it but anyone who hasn't read about external data will not so you're right about that. But at the same time, this whole paragraph even including the note is pretty short now so I don't think we need to remove it. Anyone who reads this will get the idea from the previous sentence and if they don't understand this question they'll probably just ignore it (or click on the "external outputs" link). Up to you though, please lmk.

Reply:
I don't remember why we added it. Probably had to do with reviewing or writing about the external outputs, cache, etc.

Reply:
Yep, that's expected from Ruslan :)

The problem with this that we mix so different things in the paragraph that anyone would think that this advanced one is of the same importance (while it's not). If we decide to keep, make it a paragraph, put `Advanced:`  or `in an advanced case ....`. So that it's clear for 99% of the readers it's super clear that they don't care about this.

Reply:
Ok I've removed it and placed a similar but more correct note in the actual external output doc instead. See e6c244c4

> I also did small a revision of the other nearby glossary terms.